apps/mobile/README.md

```diff
@@ -119,19 +128,51 @@ const statusMessages = {
 
 #### Extraction
 
-After finishing working on the feature, run the following command to extract strings from code
-into .po files:
+Message extraction happens automatically when you commit changes to mobile — no manual action needed.
```

Comment:
Great improvement 👍

Reply:
There's a small uncovered edge case with partial commits/amends that this might occasionally fail on. I'll fix it in a followup some time soon.